py/test.sh

```diff
@@ -11,12 +12,17 @@ cargo build -p miniconf_mqtt --example mqtt
 cargo run -p miniconf_mqtt --example mqtt &
 sleep 3 # > REPUBLISH_TIMEOUT_SECONDS
 
+MC="python -m miniconf -b localhost -d dt/sinara/dual-iir/+"
+
 python -m miniconf -b localhost dt/sinara/dual-iir/01-02-03-04-05-06 '/stream="192.0.2.16:9293"'
-python -m miniconf -b localhost -d dt/sinara/dual-iir/+ '/afe/0'       # GET
-python -m miniconf -b localhost -d dt/sinara/dual-iir/+ '/afe/0="G1"'  # SET
-python -m miniconf -b localhost -d dt/sinara/dual-iir/+ '/afe/0='      # CLEAR
-python -m miniconf -b localhost -d dt/sinara/dual-iir/+ '/afe?' '?'    # LIST-GET
-python -m miniconf -b localhost -d dt/sinara/dual-iir/+ '/afe!'        # DUMP
-sleep 1  # dump is asynchronous
+# GET SET CLEAR LIST DUMP
+$MC '/afe/0' '/afe/0="G10"' '/afe/0=' '/afe?' '?' '/afe!'
+sleep 1  # DUMP is asynchronous
+
+$MC '/four=5'
```

Comment:
Shouldn´t we test that this failed? I.e. use the same `test $? -ne 0 && exit 1`?

Reply:
The `set -e` does that for everything.

Reply:
Ah sorry, I got confused and thought the test was for <4, not >4